package/submariner.sh

```diff
@@ -20,4 +20,34 @@ sysctl -w net.ipv4.conf.all.send_redirects=0
 
 export PATH=$PATH:/usr/libexec/strongswan
 
+function find_iptables_on_host() {
```

Comment:
We probably can extract that common bit with route-agent later on a .sh which both could use.